tests/test-cases/Group6-VIC-Machine/6-07-Create-Network.robot

```diff
@@ -328,6 +337,62 @@ Bridge network - invalid bridge network range
 Bridge network - valid with IP range
     Pass execution  Test not implemented
 
+Container network port group invalid 1
```

Comment:
Instead of "invalid 1" and "invalid 2", you should just say what makes the port group name invalid.